test/unit/test_data_serialization.py

```diff
@@ -340,8 +340,9 @@ def assert_primitive_results_equal(self, primitive_result1, primitive_result2):
 
     def assert_pauli_lindblad_error_equal(self, error1, error2):
         """Tests that two PauliLindbladError objects are equal"""
-        self.assertEqual(error1.generators, error2.generators)
-        self.assertEqual(error1.rates.tolist(), error2.rates.tolist())
+        if error1 or error2:
```

Comment:
This might not be necessary for a test if it isnt one of the cases, but in general this should be
```suggestion
        if error1 and error2:
```
and then adding an `else` clause to handle case where only 1 of the errors is None